documentation/book/appendix_metrics.adoc

```diff
@@ -8,74 +8,69 @@ ifdef::InstallationAppendix[]
+=== Grafana
 
-[source,shell]
-export NAMESPACE=[namespace]
-kubectl create sa prometheus-server
-kubectl apply -f https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/master/metrics/examples/prometheus/cluster-reader.yaml
-kubectl create clusterrolebinding read-pods-binding --clusterrole=cluster-reader --serviceaccount=${NAMESPACE}:prometheus-server
+A Grafana server is necessary to get a visualisation of the Prometheus metrics.  The source for the Grafana docker image used can be found in the `./metrics/examples/grafana/grafana-openshift` directory.
 
-where `[namespace]` is the namespace/project where the Apache Kafka cluster was deployed.
+==== Deploying on {OpenShiftName}
 
-Finally, create the Prometheus service by running
+To deploy Grafana the following commands should be executed:
 
-[source,shell]
-kubectl apply -f https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/master/metrics/examples/prometheus/kubernetes.yaml
+[source,shell,subs=attributes+]
+oc apply -f https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{ProductVersion}/metrics/examples/grafana/kubernetes.yaml
 
-==== Grafana
+ifdef::Kubernetes[]
+==== Deploying on {KubernetesName}
 
-A Grafana server is necessary only to get a visualisation of Prometheus metrics.
+To deploy Grafana the following commands should be executed:
 
-To deploy Grafana on {KubernetesName}, the following commands should be executed:
+[source,shell,subs=attributes+]
+kubectl apply -f https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{ProductVersion}/metrics/examples/grafana/kubernetes.yaml
 
-[source,shell]
-kubectl apply -f https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/master/metrics/examples/grafana/kubernetes.yaml
+endif::Kubernetes[]
 
 === Grafana dashboard
 
-As an example, and in order to visualize the exported metrics in Grafana, the simple dashboard https://github.com/strimzi/strimzi-kafka-operator/blob/master/metrics/examples/grafana/kafka-dashboard.json[`kafka-dashboard.json`] file is provided.
-The Prometheus data source, and the above dashboard, can be set up in Grafana by following these steps.
+As an example, and in order to visualize the exported metrics in Grafana, two sample dashboards are provided https://github.com/strimzi/strimzi-kafka-operator/blob/{ProductVersion}/metrics/examples/grafana/strimzi-kafka.json[`strimzi-kafka.json`] and https://github.com/strimzi/strimzi-kafka-operator/blob/{ProductVersion}/metrics/examples/grafana/strimzi-zookeeper.json[`strimzi-zookeeper.json`].
+These dashboards represent a good starting point for key metrics to monitor Kafka and ZooKeeper clusters.
+Please note that they are not representative of all the metrics available.
```

Comment:
Can we link to a reference for all the metrics which are available?

Reply:
Ok.  I'll add a few references.